skipruntime-ts/ts/src/skipruntime_runner.ts

```diff
@@ -95,10 +96,10 @@ function toWriters(
 }
 
 class SimpleToGenericSkipService implements GenericSkipService {
-  tokens?: Record<string, number>;
+  refreshTokens?: Record<string, RefreshToken>;
```

Comment:
I don't understand the distinction between `Token`s and `RefreshToken`s. Each Token has a name and duration, and an array of them seems isomorphic to this `Record<string, RefreshToken>` which is like an association list right?

Reply:
`Token` here is an implementation detail of the `TimedQueue` which manages timing the token refreshes; users should only be dealing with `RefreshToken`.  I've now renamed and shuffled things around to make that more clear and avoid the name clash.

The types are indeed isomorphic but they have different interpretations: a `Token` (now `TQ_Token`) is an identifier paired with a _duration_ which the `TimedQueue` uses to schedule periodic refreshes. A `RefreshToken` on the other hand is a handle on the actual value being refreshed -- set to the current timestamp on refresh, though it should just be used as an opaque dependency in the computation graph, e.g. to trigger polls of external non-reactive dependencies.